dockerfiles/ocaml/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.10-alpine
+WORKDIR /go/src/github.com/sourcegraph/lsp-adapter
+COPY . .
+RUN CGO_ENABLED=0 GOBIN=/usr/local/bin go install github.com/sourcegraph/lsp-adapter
+
+# 👀 Add steps here to build the language server itself 👀
+# CMD ["echo", "🚨 This statement should be removed once you have added the logic to start up the language server! 🚨 Exiting..."]
+
+FROM node:9
+
+# Use tini as entrypoint to correctly handle signals and zombie processes.
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+
+USER node
+# see https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#global-npm-dependencies
+ENV NPM_CONFIG_PREFIX=/home/node/.npm-global
+ENV PATH=$PATH:/home/node/.npm-global/bin
+RUN npm i -g ocaml-language-server reason-cli@3.1.0-linux
+
+COPY --from=0 /usr/local/bin/lsp-adapter /usr/local/bin
```

Comment:
Does that mean I always have to first build the LSP adapter and put it into /usr/local/bin on the host?

Reply:
nvm, I see you build it at the top

Reply:
Yes.

Reply:
Or otherwise have some way of building the binary and executing it on the host.